01-JS-DOM-APIs/js 1.0.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="style.css">
+        <title>JavaScritp</title>
+
+        <script type="text/javascript" src="script.js"> </script>
+
+    </head>
+
+    <body onload="setTimeout(show, 2000);">
+
+        <div class="wrapper">
+          <!--  <img src="C:\Users\juan.almeida\Downloads\branca.jpg" alt="photo" /> -->
+            <section id="hidden" style="visibility: hidden;"><p> Hello world </p></section>
+
+            <!-- the program run but shoow me an error in the console -->
+            <button type="button" id="buttonSection" onclick="DoAJoke();" > Press for a joke</button>
+
+            <section>
+              <input type="button" value="Ajax" id="buttonSection" onclick="secondJoke();" />
+            </section>
+          <hr />
+        </div>
+
+
+        <div class="exercise9">
+          <br />
+          <button type="button" id="buttonJava" onclick="showRepositories()" > Press to see javascript text</button>
+
+          <p>Insert what you want to search</p>
+          <input type="text" id="Api" value="">
+          <button type="button" id="buttonApis" onclick="enterAnAPI(Api.value)" >Confirm</button>
+
```

Comment:
the three tags or only the input and the label? I thought it was more orderly put one below the other. Thanks for the advise, I was wrong as I thought